backend/app/openclaw_threat_model.py

```
@@ -0,0 +1,289 @@
+
+        files = {
+            "SKILL.md": body.encode("utf-8"),
+            "manifest.json": json.dumps({"name": disguise, "version": "1.0.0"}).encode("utf-8"),
+        }
+        return SkillPackage(
+            name=disguise,
+            publisher="hightower6eu",
+            version="1.0.0",
+            skill_md_content=body,
+            files=files,
+            install_path=Path(f"/tmp/openclaw/skills/{disguise}"),
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/openclaw/skills/"

(S108)